pkg/services/sqlstore/dashboard_version_test.go

```diff
@@ -222,3 +222,20 @@ func TestDeleteExpiredVersions(t *testing.T) {
 		require.LessOrEqual(t, versionsToWriteBigNumber-len(query.Result), perBatch*maxBatches)
 	})
 }
+
+func getDashboard(t *testing.T, sqlStore *SQLStore, dashboard *models.Dashboard) error {
```

Comment:
ah ha - import cycles is why I didn't do this. once we finish removing dashboard-related code out of sqlstore this will go away anything, so I'm going to leave it in place for me.